TAF/testScenarios/functionalTest/V2-API/support-notifications/subscription/GET-Negative.robot

```diff
@@ -0,0 +1,90 @@
+*** Settings ***
+Resource     TAF/testCaseModules/keywords/common/commonKeywords.robot
+Suite Setup  Run Keywords  Setup Suite
+...                        AND  Run Keyword if  $SECURITY_SERVICE_NEEDED == 'true'  Get Token
+Suite Teardown  Run Teardown Keywords
+Force Tags
+
+*** Variables ***
+${SUITE}          Support Notifications Subscription GET Negative Test Cases
+${LOG_FILE_PATH}  ${WORK_DIR}/TAF/testArtifacts/logs/support-notifications-subscription-get-negative.log
+${url}            ${supportNotificationsUrl}
+${api_version}    v2
+
+*** Test Cases ***
+# /subscription/all
+ErrSubscriptionGET001 - Query all subscriptions with non-int value on offset
+    When Query All Subscriptions With Invalid Offset
+    Then Should Return Status Code "400"
+    And Should Return Content-Type "application/json"
+    And Response Time Should Be Less Than "${default_response_time_threshold}"ms
+
+ErrSubscriptionGET002 - Query all subscriptions with non-int value on limit
+    When Query All Subscriptions With Invalid Limit
+    Then Should Return Status Code "400"
+    And Should Return Content-Type "application/json"
+    And Response Time Should Be Less Than "${default_response_time_threshold}"ms
+
+# /subscription/category/{category}
+ErrSubscriptionGET003 - Query subscriptions with non-existed category
```

Comment:
Shouldn't this case return 200 and empty array?

Reply:
Confirmed with @FelixTing Query APIs with non-existed data should return 404.

Reply:
Removed.